LobbyCompatibility/LobbyCompatibility.csproj

```diff
@@ -1,30 +1,101 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
+    
+    <!-- .NET Info -->
     <PropertyGroup>
         <TargetFramework>net48</TargetFramework>
```

Comment:
Should be netstandard2.1

Reply:
Currently left it as what Max had it, if they want a different framework I'll change it.